CONTRIBUTING.md

```diff
@@ -0,0 +1,643 @@
+
+In the case of DITN, its model file has no external dependencies, except for `torch` and `einops`, so we are done.
+
+### Step 2: Adding a `load` method
+
+With the architecture code in place, we can start integrating it into spandrel. The first step is to define a `load` function. This function will take a state dict, detect the hyperparameters of the architecture (=the input arguments of the architecture's Python class), and return a loaded model.
+
+We will worry about parameter detection later, for now, we'll just return a dummy model.
+
+Create a file `src/spandrel/architectures/<arch name>/__init__.py` and add the following code:
+
+```python
+from ...__helpers.model_descriptor import ImageModelDescriptor, StateDict
+from .arch.ARCH_NAME import ARCH_NAME
+
+
+def load(state_dict: StateDict) -> ImageModelDescriptor[ARCH_NAME]:
+    # default values
+    param1 = 1
+    param2 = 2
+
+    model = ARCH_NAME(
+        param1=param1,
+        param2=param2,
+    )
+
+    return ImageModelDescriptor(
+        model,
+        state_dict,
+        architecture="ARCH_NAME",
+        purpose="SR",
+        tags=[],
+        supports_half=True,
+        supports_bfloat16=True,
+        scale=1,  # TODO: fix me
+        input_channels=3,  # TODO: fix me
+        output_channels=3,  # TODO: fix me
+    )
+```
+
+Next, fill in the template:
+
+- Replace `architecture="ARCH_NAME"` with the name of your architecture.
+- Replace the references to `ARCH_NAME` with your architecture model file and class.
+- Replace `param1` and `param2` with the parameters of your architecture. Copy the default values of the parameters from the model class and pass them into the constructor (`model = ARCH_NAME(...)`).
+- Fill in the missing value for `scale`, `input_channels`, and `output_channels`.
+  - `scale`: If your architecture is a super resolution architecture, there should be a `scale`, `upscale`, or `upscale_factor` parameter. Use this value. If your architecture is not a super resolution architecture, set this value to `1`.
+  - `input_channels`: The number of input channels of the architecture. Again, there should be a parameter for this. Something like `in_nc`, `in_channels`, `inp_channels`. If there isn't leave it at 3 for RGB images.
+  - `output_channels`: The number of output channels of the architecture. Same as for `input_channels`. However, if there is no parameter for output channels, but there is one for input channels, just use the parameter for input channels.
+- Set the purpose of the architecture. If you have a restoration architecture (e.g. denoising, deblurring, JPEG), set this to "Restoration". If you have a super resolution architecture, set this to "SR" if the scale of the model is >1 and "Restoration" otherwise. We consider 1x model for SR architectures (e.g. 1x ESRGAN model) to be restoration models. (See DITN example below.)
```

Comment:
```suggestion
- Set the purpose of the architecture. If you have a restoration architecture (e.g. denoising, deblurring, or deJPEGing), set this to "Restoration". If you have a super resolution architecture, set this to "SR" if the scale of the model is >1 and "Restoration" otherwise. We consider 1x model for SR architectures (e.g. 1x ESRGAN model) to be restoration models. (See DITN example below.)
```

Reply:
> deJPEGing

I hate this. Let's got with "removing JPEG artifacts".